src/constructs/core/ssm.test.ts

```diff
@@ -18,9 +18,7 @@ describe("SSM:", () => {
       const stack = simpleGuStackForTesting({ stack: "some-stack" });
```

Comment:
Minor: I don't quite understand the purpose of these first two tests. The assertions are confusing to me. But based on the naming (`requires the scope and parameter name`) it sounds like the compiler is doing the work here and unit tests are superfluous?

Reply:
Agreed. I think addressing https://github.com/guardian/cdk/issues/1032 would remove these tests?

Reply:
Ah yep, let me raise a PR to remove that!

Reply:
cc @akash1810 : https://github.com/guardian/cdk/pull/1157 PR now raised :)